velox/functions/prestosql/aggregates/ApproxMostFrequentAggregate.cpp

```diff
@@ -506,7 +510,9 @@ void registerApproxMostFrequentAggregate(
             valueType->kind(),
             resultType,
             name,
-            valueType);
+            valueType,
+            step,
+            argTypes);
```

Comment:
nit: reorder the arguments to be `name, step, argType, resultType, valueType` to be more consistent with the lambda's parameter list.